packages/frontmatter/src/FrontmatterBlock.tsx

```diff
@@ -189,65 +190,194 @@ export function Journal({
   );
 }
 
-function LaunchButton(props: { github: string; location: string; binder?: string }) {
-  const binder = props.binder ?? 'https://mybinder.org';
+type CommonLaunchProps = {
+  github: string;
```

Comment:
should we generalize in our naming a bit more? as in:

```
git: <url>
provider: 'github'
```

thebe does that, as the intention is to support git interface/protocol.

or are we happy with
```
github: 'https://gitlab.myinstitution.org/...'
```

Also I realise that we only have `github` in the frontmatter, but I am imagining that this git info will in future come from a more dedicates `frontmatter.services` block or the like which may have non github specific fields, like the one I suggest there.

Reply:
I think we do need to change our frontmatter; GitHub should not be the only Git repo that we can define in our frontmatter. 

I would probably have frontmatter fields for `github`, `gitlab`, etc. that are normalised to a `git` field i.e. `mystmd` should know how to build canonical URLs for those providers.

Reply:
💯